src/validation.cpp

```diff
@@ -3024,7 +3028,7 @@ static bool FindBlockPos(CDiskBlockPos &pos, unsigned int nAddSize, unsigned int
                 }
             }
             else
```

Comment:
Commit 2af8b71

Care add `{}`?

Reply:
I added the braces after the code is moved to `FlatFileSeq::Allocate`.